docs/write-side.md

```diff
@@ -21,10 +21,27 @@ Aggregate state is explicitly passed to all of these functions as an argument.
 
 ## Aggregate ID
 
-Each aggregate should have a unique ID that is immutable during the aggregate's lifetime. An Aggregate ID should stay unique in the given event store, however we also recommend to keep it
-globally unique. We recommend that you use [UUID v4](https://github.com/kelektiv/node-uuid#version-4) or [cuid](https://github.com/ericelliott/cuid) to generate aggregate IDs for scalable apps.
+Each aggregate instance should have a unique immutable ID. You should generate an aggregate ID on the client and send it to reSolve with a command that creates a new aggregate:
 
-Note that you have to generate a new Aggregate ID and send it with a command that creates a new aggregate.
+```js
+import { useCommand } from 'resolve-react-hooks'
+...
+const createShoppingListCommand = useCommand(
+  {
+    type: 'createShoppingList',
+    aggregateId: uuid(),
+    aggregateName: 'ShoppingList',
+    payload: {
+      name: shoppingListName
+    },
+  },
+  (err, result) => {
+    ...
+  }
+)
+```
+
+An Aggregate ID should stay unique in the given event store across all aggregates. We also recommend to keep it globally unique. You can use [UUID v4](https://github.com/kelektiv/node-uuid#version-4) or [cuid](https://github.com/ericelliott/cuid) to generate aggregate IDs for scalable applications.
```

Comment:
is there a difference between keeping it unique across ALL aggregates and keeping globally unique?

Reply:
I think I might have phrased it wrong, The keypoint is that it should be unique for all aggregates in the given event store . If the application is designed to be scalable, it can have multiple event stores that can technically use the same ID for different things, but we recomend avoiding this.
Can I rewrite the first sentence like this:
"An Aggregate ID should stay unique across all aggregates in the given event store."?

Reply:
Yes, this is good. Do you then delete the second sentence?

Reply:
Yes, eddited as follows:
"An Aggregate ID should stay unique across all aggregates in the given event store. You can use [UUID v4](https://github.com/kelektiv/node-uuid#version-4) or [cuid](https://github.com/ericelliott/cuid) to generate aggregate IDs for scalable applications."